django_saml2_auth_lw/views.py

```diff
@@ -22,11 +22,25 @@
 from django.http import HttpResponseRedirect
 from django.utils.http import is_safe_url
 
-from rest_auth.utils import jwt_encode
-
 # default User or custom User. Now both will work.
 User = get_user_model()
 
+def jwt_encode(user):
+    """
+    This function is copy pasted from https://github.com/Tivix/django-rest-auth/blob/master/rest_auth/utils.py#L19-L29
+    to remove the need to write `from rest_auth.utils import jwt_encode`
```

Comment:
Maybe add some explanation of why why removed that dependency. Something like this:

> Removes dependency of `django-rest-auth` lib. 
> 
> `django-saml2-auth` Only used that library for JWT encode which can be easily incorporated to our source code because is an isolated function from `django-rest-auth`.